django/thunderstore/api/cyberstorm/views/team.py

```diff
@@ -0,0 +1,41 @@
+from django.db.models import Count, Sum
+from django.http import Http404, HttpRequest, HttpResponse
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from thunderstore.repository.models.team import Team
+
+from thunderstore.api.cyberstorm.serializers import CyberstormTeamSerializer, CyberstormTeamMemberSerializer
+
+
+class TeamAPIView(APIView):
+    permission_classes = []
+
+    @swagger_auto_schema(
+        responses={200: CyberstormTeamSerializer()},
+        operation_id="api.team",
+    )
+    def get(self, request: HttpRequest, team_identifier: str) -> HttpResponse:
+        try:
+            t = Team.objects.get(
```

Comment:
Should this prefetch team member info?

Reply:
ditto: https://github.com/thunderstore-io/Thunderstore/pull/846#discussion_r1235192586